docs/release-notes/highlights-2.8.0.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[release-highlights-2.8.0]]
+== 2.8.0 release highlights
+
+WARNING: This release includes a hardened default security context for Elasticsearch containers. It is highly recommended to test against a staging environment before deploying to production.
+
+[float]
+[id="{p}-280-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.8.0 of {n}. Check <<release-notes-2.8.0>> for the full list of changes.
+
+[float]
+[id="{p}-280-logstash"]
+=== Logstash support
+
+ECK 2.8.0 introduces a new Custom Resource Definition (CRD) and controller to easily deploy and orchestrate link:https://www.elastic.co/logstash/[Logstash] on Kubernetes.
```

Comment:
@robbavey @jsvd @kaisecheng 👋 please feel free to suggest anything you would like to see mentioned in the release highlights for ECK 2.8.0 regarding Logstash 🙇

Reply:
I leave it to the Logstash team to add ab bit more here, but I would at least include a link to the documentation here, to make it easier for users to start using it.

Reply:
@robbavey @jsvd @kaisecheng I would like to merge this one tomorrow, could you please add your notes. Thanks! 🙇 

> I would at least include a link to the documentation here, to make it easier for users to start using it.

There is a link to the quickstart just below. My thought was that this might be a good entry point, I'll let the Logstash team make the changes if they think it should be changed.